bot/exts/info/information.py

```diff
@@ -537,17 +549,44 @@ async def rules(self, ctx: Context, rules: Greedy[int]) -> None:
 
         # Remove duplicates and sort the rule indices
         rules = sorted(set(rules))
+        # Remove duplicate keywords
+        keywords = set(keywords)
 
         invalid = ", ".join(str(index) for index in rules if index < 1 or index > len(full_rules))
 
-        if invalid:
+        if invalid and not keywords:
             await ctx.send(shorten(":x: Invalid rule indices: " + invalid, 75, placeholder=" ..."))
             return
 
-        for rule in rules:
-            self.bot.stats.incr(f"rule_uses.{rule}")
-
-        final_rules = tuple(f"**{pick}.** {full_rules[pick - 1]}" for pick in rules)
+        final_rules = set()
+
+        for pick in rules:
+            self.bot.stats.incr(f"rule_uses.{pick}")
+            final_rules.add(
+                f"**{pick}.** {full_rules[pick - 1][0]}\n\n"
+                f"You can also invoke this rule with the following keywords: "
+                f"{', '.join(full_rules[pick -1][1])}")
+
+        for keyword in keywords:
+            for pick, rule in enumerate(full_rules):
+                if keyword not in rule[1]:
+                    continue
+
+                self.bot.stats.incr(f"rule_uses.{pick + 1}")
+                final_rules.add(
+                    f"**{pick + 1}.** {full_rules[pick][0]}\n\n"
+                    f"You can also invoke this rule with the following keywords: "
+                    f"{', '.join(full_rules[pick][1])}")
+
+        if not rules and not final_rules:
+            # This would mean that only keywords where used and no match for them was found
+            dev_contrib_channel = await get_or_fetch_channel(constants.Channels.dev_contrib)
+            meta_channel = await get_or_fetch_channel(constants.Channels.meta)
```

Comment:
You could do `f"....... <#{constants.Channels.meta}> ....."` instead to avoid importing the function (and a potential API call if it's not in the cache for some reason).